manifests/install.pp

```diff
@@ -64,6 +67,7 @@
       }
 
       # Puppet Enterprise 3.8 and ships an embedded r10k so thats all thats supported
+      # Puppet FOSS 4.2 and up also ships an embedded r10k so thats all that should be supported
```

Comment:
This comment seems wrong, its not that it ships an embedded r10k , it ships a vendored ruby install without the r10k gem (thus the need to install below) to be clear